docs/labs.md

```diff
@@ -49,3 +49,11 @@ That's it. Now should see your new counter under the header.
 ## Multiple integration managers (`feature_many_integration_managers`)
 
 Exposes a way to access all the integration managers known to Riot. This is an implementation of [MSC1957](https://github.com/matrix-org/matrix-doc/pull/1957).
+
+## Event indexing or E2EE search support using Seshat (`feature_event_indexing`)
```

Comment:
Probably "and" is more natural since Seshat supports both.

```suggestion
## Event indexing and E2EE search support using Seshat (`feature_event_indexing`)
```